apps/01PigLatin.js

```diff
@@ -7,12 +7,54 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+//'a'!=== 'A', so we want to return the string value to all lowercase, so we use string.prototype.toLowerCase so that our new word in Pig Latin will be consistent
+
+  word = word.toLowerCase();
+
+//We need to set the vowelIndex to -1 to find out if a vowel exists in the word. EX: if index of 'a' is -1, then there's no 'a' in the string; if character index > vowelIndex (-1) then a exists. Setting it to -1 updates the value of vowelIndex. we are looking for the first vowel in the string, so we need the least possible vowel index that isn't -1.
+
```

Comment:
`Setting it to -1 updates the value of vowelIndex`

@idlemindss Technically, we are not "updating" the value of **vowelIndex**. We are creating a variable named **vowelIndex** with an initial value of -1.

You might hear the phrase "initializing variables".  That's what we are doing here.